_maps/shuttles/pirate/pirate_default.dmm

```diff
@@ -1,1357 +1,2564 @@
+	id = "piratebridgebolt";
+	name = "Bridge Bolt Control";
+	normaldoorcontrol = 1;
+	specialfunctions = 4;
+	pixel_x = -21
+	},
+/turf/open/floor/iron/stairs,
 /area/shuttle/pirate)
-"aU" = (
-/obj/structure/sign/departments/engineering,
-/turf/closed/wall/mineral/plastitanium/nodiagonal,
+"le" = (
+/obj/effect/turf_decal/stripes/line,
+/obj/machinery/computer/piratepad_control,
+/obj/item/toy/plush/moth/witchwing{
+	pixel_x = -32;
+	pixel_y = 19
+	},
+/obj/machinery/light{
+	dir = 1
+	},
+/turf/open/floor/iron/tech,
 /area/shuttle/pirate)
-"be" = (
-/obj/machinery/space_heater,
-/obj/effect/turf_decal/bot,
-/obj/machinery/atmospherics/pipe/simple/supply/hidden{
+"lm" = (
+/obj/structure/chair/fancy/shuttle{
 	dir = 4
 	},
-/obj/effect/decal/cleanable/dirt,
-/obj/effect/decal/cleanable/oil,
-/turf/open/floor/plating,
+/turf/open/floor/iron/tech,
 /area/shuttle/pirate)
-"bg" = (
-/obj/effect/turf_decal/stripes/line,
+"lA" = (
 /obj/effect/decal/cleanable/dirt,
-/obj/structure/table,
-/obj/item/gun/energy/laser/retro,
-/obj/item/gun/energy/laser/retro{
-	pixel_x = -3;
-	pixel_y = 6
+/obj/effect/decal/cleanable/blood/old{
+	pixel_x = -7;
+	pixel_y = -11
 	},
-/obj/machinery/recharger,
-/obj/machinery/turretid{
-	icon_state = "control_kill";
-	lethal = 1;
-	locked = 0;
-	pixel_y = 24;
-	req_access = null;
-	req_access_txt = "180"
+/obj/structure/fans/tiny,
+/obj/machinery/door/poddoor{
+	id = "pirate_cargo_shutters"
 	},
-/turf/open/floor/pod/light,
+/turf/open/floor/plating,
 /area/shuttle/pirate)
-"bk" = (
-/obj/effect/turf_decal/stripes/line{
+"lT" = (
+/obj/effect/mapping_helpers/tile_breaker,
+/obj/machinery/atmospherics/pipe/manifold/supply/hidden{
 	dir = 1
 	},
-/obj/machinery/power/smes/engineering{
-	charge = 1e+006
+/obj/structure/cable/yellow{
+	icon_state = "4-8"
 	},
-/obj/effect/decal/cleanable/dirt,
-/obj/structure/cable/yellow,
-/turf/open/floor/plating,
+/obj/structure/cable/yellow{
+	icon_state = "2-4"
+	},
+/turf/open/floor/pod/light,
 /area/shuttle/pirate)
-"bl" = (
-/obj/effect/turf_decal/stripes/line{
-	dir = 6
+"lW" = (
+/obj/structure/table/reinforced{
+	color = "#c1b6a5"
 	},
-/obj/machinery/light/small{
+/obj/item/flashlight/lamp{
+	pixel_x = -8;
+	pixel_y = 13
+	},
+/obj/item/phone{
+	pixel_x = -6;
+	pixel_y = 3
+	},
+/obj/item/paper_bin{
+	pixel_x = 6;
+	pixel_y = 3
+	},
+/obj/item/pen{
+	pixel_x = -6;
+	pixel_y = -2
+	},
+/turf/open/floor/pod/light,
+/area/shuttle/pirate)
+"mF" = (
+/obj/structure/table/reinforced,
+/obj/machinery/chem_dispenser/drinks/fullupgrade{
+	dir = 4
+	},
+/obj/effect/turf_decal/siding/wood{
+	dir = 1
+	},
+/turf/open/floor/wood/big,
+/area/shuttle/pirate)
+"mR" = (
+/mob/living/simple_animal/cow{
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0);
+	desc = "Stolen from a cow genetics lab, she can survive in a vacuum safely, her milk is also delicious!";
+	name = "Ol Betsy"
+	},
```

Comment:
please give pirate faction to cow so it won't misfire

Reply:
fixed, thank you!